include/RE/T/TESObjectCELL.h

```diff
@@ -204,6 +204,8 @@ namespace RE
 		INTERIOR_DATA* GetLighting();
 		float		   GetNorthRotation();
 		TESForm*	   GetOwner();
+		void		   ForEachReference(std::function<bool(TESObjectREFR* a_ref)> a_fn) const;
+		void		   ForEachReferenceInRange(const NiPoint3& a_origin, float a_radius, std::function<bool(TESObjectREFR* a_ref)> a_fn) const;
```

Comment:
Surely these callbacks should be passed references instead of pointers if they're never null?

Reply:
Is it important?

Reply:
I'll just try it